velox/functions/prestosql/tests/StringFunctionsTest.cpp

```diff
@@ -1131,6 +1135,79 @@ TEST_F(StringFunctionsTest, codePoint) {
   }
 }
 
+void StringFunctionsTest::testLevenshteinDistance(
+    const std::vector<std::tuple<std::string, std::string, int64_t>>& tests) {
+  auto leftString = makeFlatVector<StringView>(tests.size());
+  auto rightString = makeFlatVector<StringView>(tests.size());
+  for (int i = 0; i < tests.size(); i++) {
+    leftString->set(i, StringView(std::get<0>(tests[i])));
+    rightString->set(i, StringView(std::get<1>(tests[i])));
+  }
+
+  auto result = evaluate<FlatVector<int64_t>>(
+      "levenshtein_distance(c0, c1)",
+      makeRowVector({{leftString}, {rightString}}));
+
+  for (int32_t i = 0; i < tests.size(); ++i) {
+    ASSERT_EQ(result->valueAt(i), std::get<2>(tests[i]));
+  }
+}
+
+TEST_F(StringFunctionsTest, levenshteinDistance) {
+  std::vector<std::tuple<std::string, std::string, int64_t>> asciiInputs{
+      {"", "", 0},
```

Comment:
The test will be easier to read and maintain if each test case was written separately:

```
ASSERT_EQ(5, levenshteinDistance("", "hello));
```